src/lib.rs

```diff
@@ -71,7 +71,11 @@ impl ManagerServer {
         connect_timeout: Duration,
     ) -> PyResult<Self> {
         py.allow_threads(move || {
-            let runtime = Runtime::new()?;
+            let runtime = tokio::runtime::Builder::new_multi_thread()
+                .worker_threads(4)
```

Comment:
LGTM, just nit: do you want this to be configurable? Or we are ok with 4 threads in any cases.